package.json

```diff
@@ -41,7 +42,8 @@
     "server-only": "^0.0.1",
     "tailwindcss": "^3.3.0",
     "typescript": "^5.4.2",
-    "typescript-eslint": "^7.2.0"
+    "typescript-eslint": "^7.2.0",
+    "husky": "^8.0.3"
```

Comment:
husky can be added as dev dependency as well. will reduce our build size.   :)

Reply:
if i'm not wrong this already in dev dependencies